main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    guitar
+//  action 1:    strum the guitar
+guitar.strum();
+//  action 2:    tweak the guitar
+guitar.tweak();
+//  action 3:    tune the guitar
+guitar.tune();
+//  2)          
+//  Noun:    tree
+//  action 1:    trim the tree
+tree.trim();
+//  action 2:    climb the tree
+tree.climb();
+//  action 3:    pour water on tree
+tree.pourWater();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    car
+//  action 1:    drive car
+car.drive();
+//  action 2:    repair car
+car.repair();
+//  action 3:    wreck the car
+car.wreck();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    keyboard
+//  action 1:    press key on the keyboard
+keyboard.pressKey();
+//  action 2:    clean keyboard
+keyboard.clean();
+//  action 3:    smash keyboard
+keyboard.smash();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    glasses
+//  action 1:    wear glasses
+glasses.wear();
+//  action 2:    prescribe glasses
+glasses.prescribe();
+//  action 3:    fold glasses
+glasses.fold();

```

Comment:
You have chosen actions that a 3rd party performs on the object. 
- [ ] revise.  Choose actions that THE OBJECT executes.